Pano/Views/ItemDetails/DetailsView.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  DetailsView.swift
+//  KampUS
+//
+//  Created by Serdar Onur KARADAĞ on 11.07.2023.
+//
+
+import SwiftUI
+import LinkPreview
+
+struct DetailsView: View {
+    
+    @AppStorage("isPreviewShows") var isPreview = true
+    
+    @State private var searchResult: String = ""
+    var item: Post
+    
+    @State private var isBookmarked: Bool = false
+    
+    var body: some View {
+        ScrollView(showsIndicators: false) {
+            VStack(alignment: .leading) {
+                Text(item.content)
+                    .padding(.vertical, 3)
+                    .padding(.horizontal, 10)
+                    .font(.system(size: 14))
+                    .background {
+                        Color(.gray).opacity(0.3)
+                    }
+                    .cornerRadius(20)
+                
+                Text(item.title)
+                
+                if isPreview {
+                    LinkPreview(url: URL(string: item.url ?? "www.google.com"))
+                        .fixedSize(horizontal: false, vertical: true)
+                } else {
+                    if item.url != nil {
+                        Link(destination: URL(string: item.url ?? "www.google.com")!) {
+                            Text("Click this and go to content.")
+                        }.padding(.vertical)
+                    }
+                }
+
+                
+                Text("Posted by \(item.owner.userName ?? "Unkown")")
```

Comment:
Not sure how localization works on SwiftUI but we may want to use `NSLocalizedString` for the texts we are using. If there is another mechanism or it is handled by default, please mention here.